SPECS/ceph/ceph.spec

```diff
@@ -971,6 +973,7 @@ env | sort
 mkdir build
 cd build
 CMAKE=cmake
+# WITH_SEASTAR is explicitly disabled to prevent numerous vendored CVEs
```

Comment:
You should also add this comment above to the "%bcond_with seastar" line.

If someone comes in and enables it, then the `-DWITH_SEASTAR=ON` below will override your explicit `-DWITH_SEASTAR=OFF` that you're adding in this PR.